tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Install BIRD
+  apt:
+    name:
+      - bird
+
+- name: Instantiate BIRD IPv4 config
+  template:
+    src: bird.conf.j2
+    dest: /etc/bird/bird.conf
+  vars:
+    bird_fact_config: "{{ bird_ipv4_config }}"
+  notify:
+    - Reload bird service if required
+
+- name: Instantiate BIRD IPv6 config
+  vars:
+    bird_fact_config: "{{ bird_ipv6_config }}"
+  template:
+    src: bird.conf.j2
+    dest: /etc/bird/bird6.conf
+  notify:
+    - Reload bird6 service if required
+
+- name: Configure bird service
+  service:
+    name: bird
+    enabled: "{{ bird_ipv4_config != [] }}"
+    state: "{% if bird_ipv4_config != [] %}started{% else %}stopped{% endif %}"
+
+- name: Configure bird6 service
+  service:
+    name: bird6
+    enabled: "{{ bird_ipv6_config != [] }}"
+    state: "{% if bird_ipv6_config != [] %}started{% else %}stopped{% endif %}"
+
+- meta: flush_handlers
+
+- name: Enable bird service
+  service:
```

Comment:
I think you're missing a condition (that `bird_ipv6_config` is not empty). `bird6` is missing, too.

Reply:
Thanks. This is now fixed.